secure_message/authentication/authenticator.py

```diff
@@ -49,7 +49,11 @@ def check_jwt(token):
 
         return {'status': "ok"}
 
-    except JWTError:
+    except (InvalidTokenError, DecodeError, InvalidAlgorithmError,
+    InvalidAudienceError, ExpiredSignatureError, ImmatureSignatureError,
+    InvalidIssuedAtError, InvalidIssuerError, ExpiredSignature,
+    InvalidAudience, InvalidIssuer, MissingRequiredClaimError,
+    InvalidSignatureError):
```

Comment:
Although not nice, I think this is acceptable. It's explicit and we're going to replace the JWT functionality with sdc-cryptography as soon as possible.

Reply:
It seems pointless though, all of those exceptions inherit from InvalidTokenError, so why not just catch that one?

Also the indention is wrong and doesn't meet PEP8 standards

Reply:
Not all of them https://github.com/jpadilla/pyjwt/blob/master/jwt/exceptions.py

Reply:
I disagree they all do, apart from InvalidKeyError which you don't catch anyway